llvm/tools/llvm-exegesis/README.md

```diff
@@ -32,6 +32,8 @@ architectures:
     e.g. pseudo instructions and most register classes are not supported.
 * MIPS
 * PowerPC (PowerPC64LE only)
+* RISCV
+  * Supported extensions: compressed, atomic, multiply-divide, initial vector instructions.
```

Comment:
could you use the formal extension names (e.g. C, A, M, ...)

Reply:
Scalar FP and bitmanip work don't they? I've tested them on HiFive Premier P550